addons/stock/models/digest.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Digest(models.Model):
+    _inherit = 'digest.digest'
+
+    kpi_stock_delivery_count = fields.Boolean('#Delivery')
```

Comment:
Shouldn't we have a space between # and Delivery ? (in which case, to propagate to other fields :) )

Reply:
Changed to Deliveries (done the same for Receipts). Actually, following a comment on the pad, I have removed all the #, that was one I had forgotten.